components/next_steps_view/__snapshots__/next_steps_view.test.tsx.snap

```diff
@@ -4,109 +4,5 @@ exports[`components/next_steps_view should match snapshot 1`] = `
 <section
   className="app__content NextStepsView"
   id="app-content"
->
```

Comment:
You may want to update the test to set the `show` state variable to `true`, otherwise this snapshot doesn't really have any meaning.

Reply:
Good point. Fixed.